web/api/mocks/data/steps.json

```diff
@@ -1,27 +1,29 @@
 [
   {
     "id": "step-1-flow-1",
-    "name": "Flow 01 Ingest Step",
+    "name": "Flow01IngestStep",
     "description": "My Step 1 description",
     "type": "ingest",
     "sourceDatabase": "",
     "targetDatabase": "staging",
     "isValid": true,
```

Comment:
As per https://wiki.marklogic.com/display/ENGINEERING/Steps+and+Step+Definitions , "sourceDatabase" and "targetDatabase" should be in "options" and I don't see  "sourceDatabase" in options for ingest step

Reply:
@srinathgit  Hm. This is an old mock object. This PR does not suppose to fix all discrepancies in DHFPROD-2096. There is also Bruce's parallel PR that got outdated because of the scope crunch. I think we are getting into messed up situation and I need to understand how to get out of this. If I will change the database option locations now in my PR it may have some more ripple effects in other UI. Need to look deeper.